Source/Orts.Simulation/Simulation/RollingStocks/MSTSWagon.cs

```diff
@@ -1453,11 +1454,13 @@ public virtual void Parse(string lowercasetoken, STFReader stf)
                     AdhesionK = stf.ReadFloat(STFReader.UNITS.None, 0.7f); if (AdhesionK <= 0) AdhesionK = 0.7f;
                     stf.SkipRestOfBlock();
                     break;
+                case "wagon(ortsslipwarningthreshold":
                 case "wagon(ortsadhesion(ortsslipwarningthreshold":
                     stf.MustMatch("(");
                     SlipWarningThresholdPercent = stf.ReadFloat(STFReader.UNITS.None, 70.0f); if (SlipWarningThresholdPercent <= 0) SlipWarningThresholdPercent = 70.0f;
                     stf.SkipRestOfBlock();
                     break;
+                case "wagon(wheelset":
```

Comment:
I don't think this can be accepted, as it is in fact creating a non-MSTS parameter which doesn't start with ORTS

Reply:
And on top of naming, looking into this further I see wheelset is its own entire _block_ with more things inside it, which isn't something I want to mess with. It's a pretty advanced feature that I don't see used much (and found no examples of being used with incorrect syntax) so, even aside from the naming convention, it does look like there's little value in allowing wheelset to exist outside of ORTSAdhesion (). This time, I am willing to give benefit of the doubt and assume that everyone using this has read the manual and knows the syntax fully.

Reply:
In fact, `ORTSAdhesion(WheelSet` is a work-in-progress feature that is not yet documented, so I don't think anyone is using it. Currently it is of no use, until a further PR finishes the work.